docs/manual/en/buildTools/Testing-with-NPM.html

```diff
@@ -21,7 +21,7 @@ <h2>The short version</h2>
 		<p>
 			If you're comfortable with node and npm,
 			<code>
-				$ npm install three --save-dev
+				$ npm install three
```

Comment:
If you want to change the command at this place, you also have to do it here:

https://github.com/mrdoob/three.js/pull/15517/files#diff-ffe4bfbdc4eb18febc0a75a0f7b68f0eR125